src/components/OurTeamCards/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import gitHubIcon from "../../images/icons/githubLogo.svg";
+import linkedInIcon from "../../images/icons/linkedInLogo.svg";
+import defaultImg from "../../images/icons/defaultImg.svg";
+import louisImg from "../../images/louisLiSmiling.jpeg";
+import ammarImg from "../../images/ammarSammourSmiling.jpeg";
+import jennetImg from "../../images/jennetHydyrovaSmiling.jpg";
+import hibaImg from "../../images/hibaMachfejSmiling.png";
+import jamilImg from "../../images/jamilBailonySmiling.png";
+import mohammadImg from "../../images/mohammadJundiSmiling.jpg";
+import "./index.scss";
+
+const OurTeamCards = () => {
+  const { t } = useTranslation();
+  const membersInfoArr = [
+    ...t("aboutUsPage.ourTeamInfo", { returnObjects: true }),
+  ];
+  const memberImagesMap = {
+    louisImg: louisImg,
+    ammarImg: ammarImg,
+    jennetImg: jennetImg,
+    hibaImg: hibaImg,
+    jamilImg: jamilImg,
+    mohammadImg: mohammadImg,
+  };
+
+  return (
+    <Container className="ourTeamInfo">
+      <h4>{t("aboutUsPage.subHeader")}</h4>
+      <Row className="justify-content-md-center">
+        {membersInfoArr.map((member) => {
+          return (
+            <div className="singleMemberInfo" key={member.name}>
+              <img
+                src={memberImagesMap[member.portraitImg] || defaultImg}
+                alt={member.name}
+                className="portraitImg"
+              />
+              <div>
+                {member.gitHubLink && (
+                  <a
+                    href={member.gitHubLink}
+                    target="_blank"
+                    rel="noopener noreferrer"
```

Comment:
I couldn't understand what noopener noreferrer, can you use something else.

Reply:
in React you cannot use `target="_blank"` without `rel="noopener noreferrer"` for security reasons